src/auto-trace.helper.js

```diff
@@ -91,7 +80,7 @@ export function appendExtraContext(error, extraContext){
  */
 export function removeAutoTraceFromErrorStack(err){
 	if(err instanceof Error && typeof err.stack === "string"){
-		err.stack = err.stack.replace(/\n.*(?:yncStacktrace|wrapObjectWithError) ?\(.*auto-trace.*/g,'');
+		err.stack = err.stack.replace(/\n.*(?:yncStacktrace|wrapObjectWithError) ?\(.*/g,'');
```

Comment:
`removeAutoTraceFromErrorStack` was not working when auto-trace was bundled, this is because the regex was trying to remove calls that were made within the `auto-trace` file when really the function is being called from within the bundled file.

Since we don't know the name of the parent file, we no longer match against the file name